com.unity.render-pipelines.universal/Shaders/Utils/FallbackError.shader

```diff
@@ -1,6 +1,51 @@
-// Shader to use as a fallback error when rendering UniversalRP materials with built-in pipeline
 Shader "Hidden/Universal Render Pipeline/FallbackError"
 {
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "RenderPipeline" = "UniversalPipeline" "IgnoreProjector" = "True" "ShaderModel" = "4.5"}
+
+        Pass
+        {
+            HLSLPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            #pragma target 4.5
+            #pragma multi_compile _ UNITY_SINGLE_PASS_STEREO STEREO_INSTANCING_ON STEREO_MULTIVIEW_ON
+            #pragma multi_compile _ DOTS_INSTANCING_ON
+            #pragma editor_sync_compilation
+
+            #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl"
+
+            struct appdata_t
+            {
+                float4 vertex : POSITION;
+                UNITY_VERTEX_INPUT_INSTANCE_ID
+            };
+
+            struct v2f
+            {
+                float4 vertex : SV_POSITION;
+                UNITY_VERTEX_OUTPUT_STEREO
+            };
+
+            v2f vert (appdata_t v)
+            {
+                v2f o;
+                UNITY_SETUP_INSTANCE_ID(v);
+                UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o);
+                o.vertex = TransformObjectToHClip(v.vertex.xyz);
+                return o;
+            }
+
+            float4 frag (v2f i) : SV_Target
+            {
+                return float4(1,0,1,1);
+            }
+            ENDHLSL
+        }
+    }
+
     SubShader
```

Comment:
Iirc correctly you need another subshader here for target 2.0 with Universal Render Pipeline tag. So there's an error when running in both SM 4.5, 2.0 for URP and one for builtin.

Reply:
This built-in RP SM 2.0 sub shader is needed to handle the case where you're trying to render URP materials without having set the URP Asset in the project settings beforehand. As agreed on Slack, I just added a comment about this to explain why this sub shader does exist.